examples/minimal/emr-cluster.tf

```
@@ -39,18 +39,57 @@ module "emr" {
   emr_ec2_iam_policy_name       = "${var.name_prefix}-ec2-policy"
   master_instance_group_name    = "${var.name_prefix}-MasterInstanceGroup"
   core_instance_group_name      = "${var.name_prefix}-CoreInstanceGroup"
-  emr_managed_master_sg_name    = "${var.name_prefix}-EMR-Master"
-  emr_managed_core_sg_name      = "${var.name_prefix}-EMR-Core"
-  emr_additional_master_sg_name = "${var.name_prefix}-EMR-Additional-Master"
-  emr_additional_core_sg_name   = "${var.name_prefix}-EMR-Additional-Core"
-  emr_service_access_sg_name    = "${var.name_prefix}-EMR-Service-Access"
+  emr_managed_sg_name           = "${var.name_prefix}-EMR-Managed"
 
   # Scale
-  master_group_instance_count = 1
-  core_group_instance_count   = 4
-  master_instance_type        = "m4.xlarge"
-  core_instance_type          = "r5.4xlarge"
-  master_ebs_size             = 50
-  core_ebs_size               = 200
-  core_bid_price              = "1.260" # r5.4xlarge on emr -> $1.008 + $0.252 = $1.260
+  master_instance_on_demand_count = 1
+  core_instance_on_demand_count   = 4
+  # core_instance_spot_count    = 4
+  # core_bid_price_as_percentage_of_on_demand_price = 100
+  master_instance_type = "m4.large"
+  core_instance_type   = "r5.4xlarge"
```

Again, not your mistake, but we've found that running spark/hbase/ganglia on the same cluster requires more memory than a m4.large for the master. `m4.2xlarge` is what we're using for our internal tests.

```suggestion
  master_instance_type = "m4.2xlarge"
  core_instance_type   = "r5.2xlarge"
```